### ISSUE TITLE
Refine MakerFox prototypes

### ISSUE THREAD

**Issue by secretrobotron** [MozillaFoundation/plan#335]
Continuing work from #267 to create 5 sketches/prototypes. Should have 5 crisp ideas/documents/prototypes.

The primary deliverable is assets for a presentation Monday, Feb 23rd that will serve to inspire those in the MoCo Steering committee meeting.

#### Deliverables

* [x] Prototype developed in #267 made into a video/animation to be included in presentation on Monday MoCo Steering meeting
* [x] Develop two more ideas into wireframes/low fidelity prototypes
* [x] Convert those (:arrow_up:) into slides

#### RACI
Phase: Research
Owner: @simonwex 
Decision: @davidascher
Lead design: @xmatthewx + @flukeout 
Lead dev: @toolness 
Quality: @thisandagain

**Comment by jgmac1106** [comment]
Not sure what the other four prototypes are for MakwrFox or if this is place to throw ideas but what mentors and webmakers need is a functional and web based whiteboard.

**Comment by thisandagain** [comment]
As per note from @davidascher we need to have a little more definition to go with the high-level slide deck that @xmatthewx produced in the last sprint by the end of the week (February 20th).

**Comment by simonwex** [comment]
Kickoff meeting is happening today, **today**; **TODAY**! 10:30 PDT in my vidyo. I'll sell you the whole seat, but you'll only need the edge!

Agenda here: https://etherpad.mozilla.org/thispadwillselfdestruct :truck: :arrow_right: :rocket: :arrow_right: :guitar:

**Comment by simonwex** [comment]
Based on our kickoff this morning:
* [ ] @toolness is going to focus the prototype ( https://github.com/xmatthewx/webmaker-addons/issues/53 )
* [ ] @toolness is going to try to slide-ify the prototype with @flukeout 's help ( https://github.com/xmatthewx/webmaker-addons/issues/55 )
* [ ] @xmatthewx is going to explore the concept of pairing multiple Firefox devices for a better experience ( https://github.com/xmatthewx/webmaker-addons/issues/56 )
* [ ] @flukeout is going to explore the visual tools concept for creating custom versions of content ( @flukeout issue link? )
*  Daily stand-ups are 0900 PDT, 1200 EDT in my vidyo room

**Comment by HPaulJohnson** [comment]
@xmatthewx and i met today to follow-up on this morning's meeting and decide on specific use cases to finalize for Monday's brand presentation:

- Atul's image save, edit, share concept
- Collaboration concept that illustrates how groups could build pages together
- Paired browser concept showing additional features from common/popular sites appearing on mobile device (ex. YouTube controls on mobile device). 

These three were chosen to show the breadth of what's possible, from the super basic and not that cool to the interesting and utilitarian to the more far fetched and attention grabbing. 

Certainly open to feedback on refining these 3 or presenting alternatives in their place. 
-

**Comment by xmatthewx** [comment]
@HPaulJohnson - it would be helpful to have some guidance on the market, the sweet spot between niche differentiation and worthwhile user base. Is our scale Adobe or Instagram? If it's in between, what's a good benchmark?

**Comment by HPaulJohnson** [comment]
@xmatthewx not sure I have the perfect answer.

I think of it as bigger than Adobe, if by Adobe you mean a tool like photoshop that is primarily aimed at professional/designers. That feels more like the Developer's Edition and this should serve a bigger market than that, something more like an audience of anyone who has ever (wanted to) edit/shar/repurpos an image.

At the same time benchmarking against Instagram (or Tumblr, etc) isn't right either, mainly because we aren't looking to build a competitor to those. Closer in scale (hopefully) but not a good analog tool-wise. 

I think the way David put it might be most helpful - the market for this is anyone using Chrome. What kind of functionality can FF + Webmaker offer to compel users to make the switch. I think the segmentation strategy is more about differentiation. We create developer browser for developers, privacy browser for people concerned about protecting themselves online, maker browser for anyone who is actively contributing/creating stuff on the web, etc.

Hope this is helpful for this exercise.

**Comment by xmatthewx** [comment]
Thanks @HPaulJohnson. It's helpful to pin this down in writing. Maybe a next step is to articulate 2 or 3 groups within that broad set, and identify a problem we can solve for them. Novelty alone won't get us to a win.

**Comment by HPaulJohnson** [comment]
Wanted to confirm the use cases we agreed on this morning that will be used for upcoming presentation:  

- @toolness  image save/edit/share concept
- @flukeout   "theme maker" concept
- @xmatthewx search box take-over concept

In addition, for presentation purposes, we can end with a slide that something like "these are just 3 of many concepts being developed" and include a reference/visual to the paired browser concept previously discussed. 

I'll share some additional thoughts about market/segment as I try to synthesize everything into a slide or 2.

**Comment by xmatthewx** [comment]
Yep. That's correct. 

 - You can grab anything useful in [this dropbox folder](https://github.com/MozillaFoundation/plan/issues/335#issuecomment-75334383) 
 - Search takeover: [home](https://www.dropbox.com/sh/2d0n1tl78focgz0/AACz72kd-dGqZrHl2s4m2LJGa?dl=0#lh:null-1_maker_b.png) and [everywhere](https://www.dropbox.com/sh/2d0n1tl78focgz0/AACz72kd-dGqZrHl2s4m2LJGa?dl=0#lh:null-1_maker_c.png)
 - Paired browser: [color collector](https://www.dropbox.com/sh/2d0n1tl78focgz0/AACz72kd-dGqZrHl2s4m2LJGa?dl=0#lh:null-4b_6%20colors.png)
 - Less quirky, but maybe useful: [the writable web](https://www.dropbox.com/sh/2d0n1tl78focgz0/AACz72kd-dGqZrHl2s4m2LJGa?dl=0#lh:null-3_everywhere_b.png)
 - thinking behind addon: the web is full of content ripe for editing and making. The first step to editing/making is inaccessible to most people. ([a couple of anecdotes](https://github.com/mozilla/webmaker-addons/issues/3#issuecomment-68929560) on things people do to connect the dots)

**Comment by simonwex** [comment]
In addition to the above image assets for slides, @toolness' great video is available here:

https://toolness-fun.s3.amazonaws.com/one-minute-image-maker-demo.mp4

**Comment by simonwex** [comment]
@HPaulJohnson can you confirm (by closing this issue) that you have everything you need for the rescheduled presentation?

**Comment by xmatthewx** [comment]
@HPaulJohnson - just a quick note that one of our concepts aligns well with the plans for v3 of FxOS and with the general effort to foster task continuity between FF on all devices. 

Compose a message, craft a post, share a doodle ... with a singular point of entry, people can create as easily as they can consume. 

![1_maker_b](https://cloud.githubusercontent.com/assets/53411/6448398/bb575a40-c0e7-11e4-9091-f20ce238dd5c.png)

**Comment by flukeout** [comment]
Link to "Themer" prototype screens - http://bit.ly/1vTEwIj

**Comment by OpenMatt** [comment]
should we move this to another milestone, so it doesn't get lost?

**Comment by HPaulJohnson** [comment]
Meeting to present these concepts is happening this Friday (3/13). I can report back with any additional feedback at that time, then we can likely close this specific issue.

**Comment by HPaulJohnson** [comment]
Meeting happened and initial prototypes were presented. 

Likely follow-up will be to explore the entire set of ideas and do some mapping back to a small handful of individual user personas/profiles. 

Closing this issue and will open a new one when specific deliverables are identified.

**Comment by simonwex** [comment]
@HPaulJohnson I'd love a report-back from the presentation. Could you schedule a session with the group?

**Comment by HPaulJohnson** [comment]
Here is the MakerFox portion of the deck we shared: https://docs.google.com/a/mozillafoundation.org/presentation/d/17fgbcOLhhTHoNDUmO2cIiHiFRTeWWHVZPZj5hoOM37s/edit#slide=id.g8da6c5f45_054

Will find some time to reconvene on next steps when I confirm @davidascher schedule (I'm on PTO Wed-Fri this week). Either in the next two days or early next week. 

TL;DR is that next step is to connect use case/prototypes more closely with specific "maker" personas. Prototypes appealed to very broad audience (like, anyone who uses Chrome) and Eric, in particular, would like to see a bit more definition between specific segemented audience and particular use cases/features in a next round.
